analyze.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import argparse
+from pathlib import Path
+import sys
+
+# path to all benchmark modules
+sys.path.append("analyze/")
+if sys.version_info[0] < 3:
+    raise "Python 3 should be used to run this script"
+
+import config
+from timestamp import *
+from benchcases import *
+
+# handle params
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument("filename", help="File with timestamps")
+arg_parser.add_argument("-c", "--case", type=BenchmarkCase,
+                        choices=list(BenchmarkCase),
+                        help="Specify the benchmark case you want to analyze")
+args = arg_parser.parse_args()
+
+try:
+    if not (args.filename and Path(args.filename).is_file()):
+        raise RuntimeError("Wrong path to timestamp dump file")
```

Comment:
What about changing the error message slightly? `"Cannot find file: {}.".format(args.filename)"`

Reply:
@jbech-linaro, thanks, will fix